qiita_pet/templates/redbiom.html

```diff
@@ -0,0 +1,167 @@
+{% from qiita_core.qiita_settings import qiita_config %}
+{% from future.utils import viewitems %}
+{% extends sitebase.html %}
+
+{%block head%}
+<link rel="stylesheet" href="{% raw qiita_config.portal_dir %}/static/vendor/css/select2.min.css" type="text/css">
+<script type="text/javascript" src="{% raw qiita_config.portal_dir %}/static/vendor/js/select2.min.js"></script>
+<script type="text/javascript" src="{% raw qiita_config.portal_dir %}/static/js/sharing.js"></script>
+<script type="text/javascript">
+  $(document).ready(function() {
+    moi.init(null, window.location.host + '{% raw qiita_config.portal_dir %}/study/list/socket/', function(){}, error, error);
+    moi.add_callback('sel', show_alert);
+    $("#redbiom-table").dataTable({
+      "dom": 'lfrt<"footer">ip',
+      'footerCallback': function ( row, data, start, end, display ) {
+        var api = this.api(), data;
+        var total_samples = api
+          .column(4)
+          .data()
+          .reduce( function (a, b) { return a + b.samples.length; }, 0);
+
+        $('.footer').addClass("col-md-12 text-right");
+        if (total_samples == 0) {
+          text = ''
+        } else {
+          text = 'Found ' + total_samples + ' samples.';
+        }
+        $('.footer').html(text)
+      },
+      "columns": [
+        { "data": null, "orderable": false, "width": "10%" },
+        { "data": null, "width": "40%"  },
+        { "data": "aname", "width": "20%"  },
+        { "data": null, "width": "20%"  },
+        { "data": null, "width": "10%" }
+      ],
+      columnDefs: [
+        {type:'natural', targets:[1,2,3,4]},
+        // render Add to Analysis button
+        {"render": function ( data, type, row, meta ) {
+          {% if current_user is not None %}
+            var text = '<input type="button" class="btn btn-sm" value="Add to analysis" onclick="redbiom_send_to_moi(' +
+                       row.artifact_id + ', ' + meta.row + ')">';
+          {% else %}
+            var text = ''
+          {% end %}
+          return text;
+        }, targets: [0]},
+        // render Study title
+        {"render": function ( data, type, row, meta ) {
+          {% if current_user is not None %}
+            var text = '<a target="_blank" href="{% raw qiita_config.portal_dir %}/study/description/' +
+                       row.study_id + '" role="button">' + row.study_title + ' (ID: ' + row.study_id + ')</a>';
+          {% else %}
+            var text = row.study_title + ' (ID: ' + row.study_id + ')';
+          {% end %}
+          return text;
+        }, targets: [1]},
+        // render Artifact Processing
+        {"render": function ( data, type, row, meta ) {
+            var text = '';
+            if (!(!row.command)) {
```

Comment:
yup, again read somewhere that to test is a variable is not undefined or null the easiest is to do !row.command but didn't want to add an else so just negated that boolean ...